xds/src/main/java/io/grpc/xds/EnvoyServerProtoData.java

```diff
@@ -380,114 +316,30 @@ public String toString() {
   static final class FilterChain {
     // TODO(sanjaypujare): flatten structure by moving FilterChainMatch class members here.
     private final FilterChainMatch filterChainMatch;
+    private final HttpConnectionManager httpConnectionManager;
```

Comment:
I don't see this being consumed anywhere. If this is for some future enhancement, can you add a comment to that effect?

Reply:
Yes, this PR mainly populates the per-FilterChain HttpConnectionManager for the TCP listener used by server side. The PR I am currently working on will take use of the HttpConnectionManager here (for HttpFilter and RDS).